projects/ideabox-triples.md

```diff
@@ -220,4 +227,8 @@ You will need the `svg` files for the delete, upvote, and downvote icons. [Here'
   * [ ]  There are less than 3 global variables
   * [ ]  There are no nested if else statements
 * [ ]  Exceptional - Application adds to code quality by refactoring all for loops into the proper array prototype iteration methods and:
-  * [ ] Using logical operators instead of if/else
+  * [ ]  Using logical operators instead of if/else
+  * [ ]  Application changes anonymous functions to arrow functions and changes all variables to be block scoped (`let` and `const`)
+  * [ ]  Application stores all ideas in one array in local storage, instead of individually
+  * [ ]  All ideas that do not need to be shown on the dom should be completely removed from the dom, instead of being hidden from view
```

Comment:
Should we specify, we are looking for this when the user is searching through the ideas?